### ISSUE TITLE
推理速度FPS问题

### ISSUE THREAD

**Issue by Overtrued** [PaddlePaddle/PaddleDetection#8764]
### 问题确认 Search before asking

- [X] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### 请提出你的问题 Please ask your question

背景：我有两个数据集A和B。两个数据集都是2000张。A为200x200的，B为4010x1024的图像。使用PP-detection里面的SSD、Faster-RCNN 和PPyoloE
问题：1. 我两个数据集使用相同配置，A数据集推理速度3个小时就完成了，而B需要3-4days。为什么相差这么大呢？2 . 使用yolo系列，速度都是3-4小时。
    很大的疑惑。 我需要怎么配置文件训练数据集B呢。

**Comment by Overtrued** [comment]
基于3080 。
问题fps ：[12/20 09:18:28] ppdet. engine INF0: Epoch: [1][ 20/223] tearning rate: 0.000657 Loss: 14,761235 eta: 2 days, 18:28:20 batch cost: 3.6932 data cost: 3.3984 1ps: 2.1662 1mages/s
[12/28 89:20:04] ppdet. engine INFD: Epoch: [1] [ 48/223] learning rate: 8.008684 loss: 12.849310 eta: 2 days, 18:48:40 batch cost: 4.8161 data cost: 4.4678 1ps: 1.6611 inages/s
[12/20 09:21:33] ppdet. engine INF0: Epoch: [1] 1 60/223] learning rate: 0.000711 1oss: 13.937094 eta: 2 days, 18:43:33 batch cost: 4.4585 data cost: 4.1095 1ps: 1.7943 images/s
[12/20 09:23:01] ppdet. engine INF0: Epoch: [1] [ 80/223] learning rate: 0.000737 Loss: 18.032242 eta: 2 days, 18:35:47 batch cost: 4.4050 data cost: 4.0398 1ps: 1.8161 1nages/s
 正常A：
ne INF0: Epoch: [0] [ 60/180] learning rate: 0.000413 loss: 8.783234 eta: 3:35:43 batch cost: 0.1605 data cost: 0.0477 ips: 49.8095
 insges/sne INF0: Epoch: [0] [ 80/180] learning_rate: 0.000440 1oss: 7.820248 eta: 3:09:45 batch cost: 0.1543 data_cost: 0.0439 1ps: 51.8586 1nages/s
ne INF0: Epoch: [0] [100/188] tearning rate: 3.000467 1oss: 7.287611 eta: 2:54:30 batch cost: 0.1574 data cost: 0.0465 ips: 58.8124 1nages/s
ne INF0: Epoch: [0] [120/188] Learning rate: 8.080493 Loss: 7.191412 ota: 2:42:46 batch cost: 0.1448 data cest: 0.0357 ips: 55.2676 1nages/s
ne INF0: Epoch: [0] [140/188] learning rate: B.080520 loss: 7.356677 eta: 2:36:48 batch cost: 0.1693 data cost: 0.0489 ips: 55.3915 inages/s
ne INF0: Epoch: [0] [160/180] learning_rate: 0.000547 loss: 7.093461 eta: 2:29:06 batch_cost: 0.1524 data_cest: 0.0435 Ips: 52.4829 insges/s
8. checkpoint INFO: Save checkpoint: output ne INFO: Evat iter: 0
ne INFD: Eval iter: 188ng INFO: Eval iter: 280ne INFO: Eval iter; 300
ics. netrics INFD: Accumulating evalustation results...
1cs. netrics INFO: AP(0.50, 11point) 4.54%
ne INFO: Tota sample nuтber: 360, average FPS: 54.134827122169725
ne INFO: Best test bbox ap is 8.045.
5. checkpoint INFO: Save checkpoint: output ne INF0: Epoch: [1] [ 0/180] learningrate: 0.000573 loss: 7.245394 eta: 2:24:21 batch cost: 0.1488 data cost: 0.0402 ips: 53.7719 inages/s

**Comment by Overtrued** [comment]
Please

**Comment by lyuwenyu** [comment]
有可能熟  第一个epoch的模型效果不好 -> 后处理候选框太多 -> nms慢
